samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -95,6 +95,7 @@ Button 1:
    .. note::
        The advertising is disabled while the Fast Pair Provider is connected to a Bluetooth Central.
        The discoverable advertising is automatically switched to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.
+       The advertising mode is automatically switched to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.
```

Comment:
This new line reads like contradicting line 96, because "while" also includes case now added in line 98. What about:
```suggestion
       The advertising mode automatically switches to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.
       If the discoverable advertising is toggled, it automatically switches to the not discoverable advertising with the show UI indication mode after a 10-minute timeout.
       
```

Reply:
The 96 line is still true. Advertising is disabled and the advertising mode is switched. Of course, the only consequence of switching advertising mode while advertising is disabled (device is connected) is changing the led effects. Actual advertising in new mode starts after the device disconnects.